chapter_computer-vision/index.md

```diff
@@ -1,8 +1,8 @@
 # Computer Vision
 
-Many applications in the area of computer vision are closely related to our daily lives, now and in the future, whether medical diagnostics, driverless vehicles, camera monitoring, or smart filters. In recent years, deep learning technology has greatly enhanced the performance of computer vision systems. It can be said that the most advanced computer vision applications are almost inseparable from deep learning.
+Many applications in the area of computer vision are closely related to our daily lives, now and in the future, whether medical diagnostics, driverless vehicles, camera monitoring, or smart filters. In recent years, deep learning technology has greatly enhanced computer vision systems performance. It can be said that the most advanced computer vision applications are nearly inseparable from deep learning.
```

Comment:
I would prefer "the performance of computer vision systems" here.

Reply:
I would defer to an independent editor as far as use of the indefinite article in the book, as my general impression is it is perhaps overused. 
However, of no major consequence.
Discussing Quirk et al., "A Comprehensive Grammar of the English Language", 5.58, The articles with abstract noncount nouns is perhaps best left for a later date as it is hardly on top of the backlog for this project.

Reply:
"computer vision systems performance" should be changed to "computer vision systems' performance"

"the performance of computer vision systems" emphasizes performance here.